3Box/3box-profile-hover.md

```diff
@@ -0,0 +1,210 @@
+
+![Tile](https://github.com/Solexplorer/Content/blob/master/3Box/images/tile.png)
+
+The `hover` element appears once the `tile` element is hovered and it pulls the data from the user's 3Box profile such as name, image, and description.
+
+![Hover](https://github.com/Solexplorer/Content/blob/master/3Box/images/hover.png)
+
+As outlined in the previous section the profile-hover uses the profiles API to get the data but if the user didn't sign up with 3Box it shows the shortened address and a blocky identicon like this:
+
+![No_account](https://github.com/Solexplorer/Content/blob/master/3Box/images/no_account.png)
+
+## Using profile hovers in a React app
+
+Before diving in, you will need to have NodeJS installed, follow the guide
+[here](https://nodejs.org/en/download/package-manager/)
+
+As a first step, you will need to create a React application in which we will use
+profile hovers.
+
+`npx create-react-app profile-demo`
+
+Afterward, let's install the `profile-hover` component.
+
+```
+cd profile-demo
+npm i -S profile-hover
+```
+
+Let's open the `App.js` and import the package:
+
+`import ProfileHover from 'profile-hover';`
+
+At this point, we can use the imported component to our liking. There are various
+ways with which we can customize how to display it.
+
+Let's modify the `App.js` in order to see how easy it is to use this feature.
+
+```javascript
+import React from 'react';
+import "./App.css";
+import ProfileHover from 'profile-hover';
+
+const App = ({}) => {
+  return (
+    <>
+      <div className='ethAddress'>
+        <ProfileHover orientation="right" url="www.kauri.io" showName tileStyle address='0x3f46680099cf623163c96747a8addb85a1da1cd1' />
+      </div>
+
+      <div className='ethAddress'>
+        <ProfileHover orientation="right" noImgs showName tileStyle address='0xec1f83cf6a6dc7ee04a79c99a67cd3800111b355' />
+      </div>
+      <div className='ethAddress'>
+        <ProfileHover orientation="right" noCoverImg showName tileStyle address='0x499c9c826a356f72926a258afa63bcdb4df33702' />
+      </div>
+      <div className='ethAddress'>
+        <ProfileHover orientation="right" noCoverImg showName tileStyle address='0xfc27bf5bdc304ea27e74fd3f514e5400f0a9e76d' />
+      </div>
+      <div className='ethAddress'>
+        <ProfileHover orientation="right" noImgs showName address='0x781901682d3e9341a6c195ee6fe58047a9235f07' />
+      </div>
+      <div className='ethAddress'>
+        <ProfileHover orientation="right" displayFull showName address='0x442dccee68425828c106a3662014b4f131e3bd9b
+' fullDisplay />
+      </div>
+      <div className='ethAddress'>
+        <ProfileHover orientation="right" showName address='0xd75bb1e6ef53d0c65bcf53ecb19d8c64a7026c58' noTheme>
+          0xa8ee0babe72cd9a80ae45dd74cd3eae7a82fd5d1
+        </ProfileHover>
+      </div>
+    </>
+  )
+}
+
+export default App; 
+```
+
+Let me explain the example above, `ProfileHover` is the React component used to
```

Comment:
You've used "you" and "we" to talk to the reader so far, stick to one, or it's confusing. So far you've mostly used "we", so stick to that.